prow/kube/prowjob.go

```diff
@@ -99,43 +48,3 @@ const (
 	// carries the PR number associated with the job, eg 321.
 	PullLabel = "prow.k8s.io/refs.pull"
 )
```

Comment:
move these into types.go? or into prowjob types?

Reply:
I wanted this change to be a NOOP so we can do that in a follow-up